src/main/webapp/script.js

```diff
@@ -38,6 +38,87 @@ function displayError() {
   alert(alertMessage);
 }
 
+//Array of the (currently 6 for this demo build) 15 listings gathered from the fetch request
+let listingsArray = [];
+
+//Count of the total businesses in the fetch request, used to set a unique id for each card
+let listingCardCount = 0;
```

Comment:
nit: Just a suggestion: I'd rename this "totalCardCount" specifically based on how it's being used.

Reply:
Done